modules/tests/photons_control_tests/device_finder/test_device.py

```diff
@@ -57,12 +86,31 @@ def device(self):
             "kelvin": 2500,
             "firmware_version": "1.2",
             "product_id": 22,
-            "cap": ["multizone", "color"],
+            "product_name": "LIFX Color 1000",
+            "product_type": "light",
+            "cap": [
```

Comment:
https://github.com/delfick/photons/blob/main/modules/photons_pytest.py#L504

Reply:
I think I discoved this after manually changing all the test devices and then forgot to go back to see if it would solve my previous problems.